sdk/communication/Azure.Communication.Sms/tests/samples/Sample1_SmsClient.cs

```diff
@@ -65,5 +83,41 @@ public async Task SendingGroupSMSMessageWithOptions()
             }
             #endregion Snippet:Azure_Communication_SmsClient_Send_GroupSmsWithOptions
         }
+
+        [Test]
+        public async Task SendMessageTroubleShooting()
+        {
+            SmsClient smsClient = CreateSmsClient();
+            #region Snippet:Azure_Communication_Sms_Tests_Troubleshooting
+            try
+            {
+                Response<IEnumerable<SmsSendResult>> response = await smsClient.SendAsync(
+                   //@@ from: "<from-phone-number>" // Your E.164 formatted phone number used to send SMS
+                   //@@ to: new string [] {"<to-phone-number-1>", "<to-phone-number-2>"}, // E.164 formatted recipient phone number
+                   /*@@*/ from: TestEnvironment.FromPhoneNumber,
+                   /*@@*/ to: new string[] { TestEnvironment.ToPhoneNumber, TestEnvironment.ToPhoneNumber },
+                message: "Weekly Promotion!",
+                options: new SmsSendOptions(enableDeliveryReport: true) // OPTIONAL
+                {
+                    Tag = "marketing", // custom tags
+                });
+                IEnumerable<SmsSendResult> results = response.Value;
+                foreach (SmsSendResult result in results)
+                {
+                    if (result.Successful)
+                    {
+                        Console.WriteLine($"Successfully sent this message: {result.MessageId} to {result.To}.");
+                    } else {
+                        Console.WriteLine($"Something went wrong when trying to send this message {result.MessageId} to {result.To}.");
+                        Console.WriteLine(result.ErrorMessage);
```

Comment:
Maybe we can log the status code as well?

Reply:
Sure! I can add that as well.